File: js/login_test.js
var webdriver = require('selenium-webdriver'),
    By = webdriver.By,
    until = webdriver.until,
    test = require('selenium-webdriver/testing'),
    should = require('should');

test.describe('Log in', function() {
    var driver;

    test.before(function() {
	driver = new webdriver.Builder()
	    .forBrowser('chrome')
	    .build();
    });

    test.it('should display notice', function() {
	driver.get('http://localhost/litecart/admin');
	driver.findElement(By.name('username')).sendKeys('admin');
	driver.findElement(By.name('password')).sendKeys('admin');
	driver.findElement(By.name('login')).click();
	return driver.findElement(By.className('notice success'))
	    .getText()
	    .should.be.eventually.equal('You are now logged in as admin');
    });

    test.after(function() {
	driver.quit();
    });
});

File: ruby/login_test.rb
require 'rspec'
require 'selenium-webdriver'

describe 'login' do
  before(:each) do
    @driver = Selenium::WebDriver.for :chrome
  end

  it 'should show login message' do
    @driver.navigate.to 'http://localhost/litecart/admin/'
    @driver.find_element(:name, 'username').send_keys('admin')
    @driver.find_element(:name, 'password').send_keys('admin')
    @driver.find_element(:name, 'login').click

    notice_text = @driver.find_element(:css, '.notice.success').text
    expect(notice_text).to eq('You are now logged in as admin')
  end

  after(:each) do
    @driver.quit
  end
end
